yolov5/val.py

```diff
@@ -156,6 +157,60 @@ def process_batch(detections, labels, iouv):
     return torch.tensor(correct, dtype=torch.bool, device=iouv.device)
 
 
+def get_current_time():
+    current_time = datetime.now()
+    current_time_str = current_time.strftime("%Y-%m-%d %H:%M:%S")  # Format the datetime as a string
+    return current_time_str
+
+
+def exception_handler(func):
```

Comment:
What about moving this to its own file?

Reply:
Ga ik doen:

from baas_utils.error_handling import exception_handler